source/construct-relators.rq

```diff
@@ -1,16 +1,16 @@
 prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
-prefix owl: <http://www.w3.org/2002/07/owl#>
 prefix skos: <http://www.w3.org/2004/02/skos/core#>
 prefix bf2: <http://id.loc.gov/ontologies/bibframe/>
+prefix madsrdf: <http://www.loc.gov/mads/rdf/v1#>
 prefix : <https://id.kb.se/vocab/>
 
 construct {
     ?idrel a bf2:Role;
         skos:notation ?notation;
         ?p ?o;
-        owl:equivalentProperty ?locrel;
+        skos:exactMatch ?locrel;
         rdfs:domain ?domain;
-        rdfs:subPropertyOf ?superrel .
```

Comment:
Why drop this?

Reply:
As with 'skos:notation',  'rdfs:subPropertyOf' is not available at https://id.loc.gov/vocabulary/relators.nt, so this code was not working. Hence nothing is dropped from a user perspective. However, I guess it would be nice to be able to link to the relators' superclasses, as was intended. In that case we need to get another relators dump or somehow query the individual relators to fetch this information(?).

Reply:
Yes, I see. I can have a go at getting more of the data from their zipped dump (in the new branch https://github.com/libris/definitions/compare/feature/augment-loc-linking which also includes a modest speedup to the selection which we could cherry-pick if we need it now).